front/app/containers/Admin/flexible-pages/routes.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PageLoading from 'components/UI/PageLoading';
+import moduleConfiguration from 'modules';
+
+const CustomNavbarContainer = React.lazy(
```

Comment:
Not sure if I'm following: should the custom navbar items be in the core (accessible for all pricing plans)?

Reply:
We should still show the list of pages, but indeed, the name `CustomNavbarContainer` is not proper anymore. I think I should rename it together with `NavigationSettings`.